@navikt/core/css/tooltip.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  --navds-tooltip-color-text: var(--navds-color-text-inverse);
+  --navds-tooltip-color-background: var(--navds-color-gray-90);
+  --navds-tooltip-color-border: transparent;
+  --navds-tooltip-color-shadow: var(--navds-color-gray-20);
+}
+
+.navds-tooltip {
+  z-index: var(--navds-z-index-popover);
+  background-color: var(--navds-tooltip-color-background);
+  color: var(--navds-tooltip-color-text);
+  box-shadow: 0 2px 4px 0 var(--navds-tooltip-color-shadow);
+  border: 1px solid;
+  border-color: var(--navds-tooltip-color-border);
+  border-radius: 4px;
+  padding: var(--navds-spacing-2);
+  white-space: nowrap;
+}
+
+.navds-tooltip--hidden {
+  display: none;
+}
+
+.navds-tooltip__wrapper {
+  position: relative;
+  display: inline-flex;
+  justify-content: center;
+}
+
+/* Tooltip should never get focus, so ignoring styling it */
```

Comment:
Hvorfor trenger vi en stil om den aldri får fokus?

Reply:
Var sikkert en tanke bak hvorfor den er der, men ser ingen grunn for det nå 🤷

Reply:
Ble satt siden tooltip hadde tabIndex={-1}, men er fjernet nå